### ISSUE TITLE
help needed: missing stylesheets

### ISSUE THREAD

**Issue by ngi32487** [selvera/npm-datepicker#2]
I managed to include your datepicker into my angular 6 project (after some necessary modifications in your source files due to new rxjs behaviour; refer to [](url) https://github.com/ReactiveX/rxjs/blob/master/MIGRATION.md, but the stylesheets seem to be missing:
![grafik](https://user-images.githubusercontent.com/39375526/40179578-fb9072e2-59e4-11e8-96c0-b71c85852d48.png)

The package @coachcare/datepicker includes the file "_theming.scss". I added this file to the styles array in angular.json, but without success... Any advice?

**Comment by matheo** [comment]
Hi @ngi32487!
I'm sooo sorry I didn't read this before!
I was in the middle of a travel a month ago, sorry!

I hope you figured that this Material module needs to be themed as the others, next to the inclusion of the Angular Material theming of your application. Like in your `styles.scss`:
```
@import '../node_modules/@coachcare/datepicker/theming';

@include mat-datepicker($theme);
```

If you need to understand the theming better read this article:
https://blog.thoughtram.io/angular/2017/05/23/custom-themes-with-angular-material.html

P.S. I just synced this datepicker to the most recent code on the official module, but with time support. Do you mind to submit a PR with your `rxjs` updates?
Thanks!! and apologies again!

**Comment by ngi32487** [comment]
Thank you for the reply, but I don't get the theme to work... maybe I have to learn scss first or port my project to use scss instead of css.

No I don't mind sharing my rxjs updates, but actually your latest release made nearly all my changes obsolete. v0.9.0 requires only minor changes:

`years-view.d.ts`:

Replace
> import 'rxjs/add/observable/fromEvent';
import 'rxjs/add/observable/of';
import 'rxjs/add/operator/mergeMap';
import 'rxjs/add/operator/sampleTime';
import { Subscription } from 'rxjs/Subscription';

with
> import { Subscription, of, fromEvent } from 'rxjs';
import { mergeMap, sampleTime } from 'rxjs/operators';

`fesm5/coachcare-datepicker.js`:

Replace
> import { Subject, merge, Subscription, of } from 'rxjs';
...
import 'rxjs/add/observable/fromEvent';
import 'rxjs/add/observable/of';
import 'rxjs/add/operator/mergeMap';
import 'rxjs/add/operator/sampleTime';
import { Observable } from 'rxjs/Observable';

with
> import { Subject, merge, Subscription, Observable, of, fromEvent } from 'rxjs';
import { mergeMap, sampleTime } from 'rxjs/operators';

...that's it.

**Comment by matheo** [comment]
@ngi32487 I've just released `@coachcare/datepicker 0.9.2` with some default `css` files like Angular Material :D

Now you can include in your `angular.json` the style directly like
```
"styles": [
  "../node_modules/@coachcare/datepicker/prebuilt/indigo-pink.css",
  "styles.css"
]
```
and get your styled datepicker.

I'm also using CSS variables supported by most of the modern browsers nowadays (https://www.w3schools.com/css/css3_variables.asp), and you can define the colors with:
```
body {
  --primary: rgba(73, 200, 242, 1);
  --primary-contrast: #fff;
  --primary-a60: rgba(73, 200, 242, 0.6);
  --primary-a80: rgba(73, 200, 242, 0.8);
}
```
Enjoy!
